pkg/image/reference/reference.go

```diff
@@ -62,6 +63,19 @@ func Parse(spec string) (DockerImageReference, error) {
 	return ref, nil
 }
 
+// GetSpecDigestTag returns the image digest from a DockerImageReference string.
+// example '@sha:22204fa2...' in an image 'registry/ocp/release@sha:22204fa2...'
+func GetSpecDigestTag(spec string) (string, error) {
+	namedRef, err := reference.ParseNamed(spec)
+	if err != nil {
+		return "", err
+	}
+	if named, ok := namedRef.(reference.Canonical); ok {
+		return fmt.Sprintf("@" + named.Digest().String()), nil
```

Comment:
Why do you need this '@' prefix?

Reply:
right, i'm not sure about that.. we can add that in a fmt.Sprintf here instead: https://github.com/openshift/oc/pull/395/files#r414557373

Reply:
I updated, func is now 'GetSpecDigest' and will add the "@" when needed in oc.